packages/monaco/src/browser/monaco-editor.ts

```diff
@@ -84,6 +84,9 @@ export class MonacoEditor extends MonacoEditorServices implements TextEditor {
     readonly onLanguageChanged = this.onLanguageChangedEmitter.event;
     protected readonly onScrollChangedEmitter = new Emitter<void>();
     readonly onEncodingChanged = this.document.onDidChangeEncoding;
+    // eslint-disable-next-line no-null/no-null
+    protected readonly onResizeEmitter = new Emitter<Dimension | null>();
+    readonly onResized = this.onResizeEmitter.event;
```

Comment:
I think [the guidelines](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#event_names) would likely prefer `onDidResize` unless there's a conflict with that name?

Reply:
No problem, I've renamed this value and squashed the commits